recipes/tikz

```diff
@@ -1 +1,2 @@
-(tikz :repo "emiliotorres/tikz" :fetcher github)
+(tikz :repo "emiliotorres/tikz" :fetcher github
+	  :files ("*.el" "templates"))
```

Comment:
```suggestion
	  :files (:defaults "templates"))
```
`:defaults` might be a better choice here since it automatically ignores -test.el files and the like.